tuning/trainercontroller/operations/hfcontrols.py

```diff
@@ -10,6 +10,8 @@
 # Local
 from .operation import Operation
 
+logger = logging.getLogger()
```

Comment:
Is this the standard way logging is being initialized elsewhere? Is there no logger name given?

Reply:
Added `__name__`. Addressed.